patches/api/0393-Add-custom-destroyerIdentity-to-sendBlockDamage.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheFruxz <cedricspitzer@outlook.de>
+Date: Sat, 26 Mar 2022 11:48:30 +0100
+Subject: [PATCH] Add custom destroyerIdentity to sendBlockDamage
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index f954801f6a5d465b8545e75d7ff5af0352d6ec0d..97b85f312713d0f7f041d812b41b30c49fc466fb 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -580,6 +580,17 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     public void sendBlockDamage(@NotNull Location loc, float progress);
+ 
+     // Paper start
++
++    /**
++     * Send block damage. This fakes block break progress for a user at a certain location.
++     * This will not actually change the block's break progress in any way.
++     *
++     * @param loc the location of the damaged block
++     * @param progress the progress from 0.0 - 1.0 where 0 is no damage and 1.0 is the most damaged
++     * @param destroyerIdentity the entity id of the destroyer
++     */
++    public void sendBlockDamage(@NotNull Location loc, float progress, int destroyerIdentity);
```

Comment:
Perhaps some additional javadoc comment should be added to the original sendBlockDamage telling them that the destroying will be this player.

Reply:
If I want to change the documentation of this method of the interface, should I move the whole method to the // Paper start part?

Reply:
no, just don't add a // Paper comment. javadoc changes don't require a comment for them.

Reply:
Added javadoc comments